server/__tests__/suites/integration/groups.test.ts

```diff
@@ -1207,6 +1207,54 @@ describe('Group API', () => {
       expect(onMembersJoinedEvent).not.toHaveBeenCalled();
       expect(onMembersRolesChangedEvent).not.toHaveBeenCalled();
     });
+
+    it('should edit group role order', async () => {
+      const response = await api.put(`/groups/${globalData.testGroupOneLeader.id}`).send({
+        verificationCode: globalData.testGroupOneLeader.verificationCode,
+        roleOrders: [
+          { role: 'member', index: 1 },
+          { role: 'air', index: 2 }
+        ]
+      });
+
+      expect(response.status).toBe(200);
+
+      const response2 = await api.get(`/groups/${globalData.testGroupOneLeader.id}`).send();
```

Comment:
no need for empty `send`.

Also, we should be testing if the order of members is as expected (member -> air)

And we should also test the case that no roleOrders are available, it sorts privileged roles first (see group utils)

Reply:
the order seems like a consumer concern, right? feel like we should just return the set and let the client sort based on index? could be off on that tho

Reply:
done